app/css/Form.scss

```diff
@@ -4,3 +4,12 @@
 .is-invalid-input {
   border-color: $danger;
 }
+
+@keyframes spinner {
+  to {transform: rotate(360deg);}
```

Comment:
I think font awesome has class that include spinning: `fa-spin`
(if I remember correctly)

Reply:
Yeah I remember seeing that indeed.